plugins/promdump.yaml

```
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: promdump
+spec:
+  version: v0.2.2
+  homepage: https://github.com/ihcsim/promdump
+  shortDescription: promdump dumps the head and persistent blocks of Prometheus.
```

```suggestion
  shortDescription: Dumps the head and persistent blocks of Prometheus.
```
No need to repeat the plugin name.

---

Although your current description works, what do you think about talking about the problem it solves instead. For example:

> Helps to move Prometheus TSDB data between clusters

---

I don't want folks to get the idea that this is a migration or backup/restore tool, because there are still things around restoring the WAL at the TSDB level that I don't understand. At the moment, I see this plugin as a SRE tool to copy data from one Prometheus instance to another. promdump itself isn't limited to just K8s.

---

As the tool matures and evolves, I can see this changing.

---

The promdump README has a [Limitation](https://github.com/ihcsim/promdump#limitations) section describing what it is currently good for, and not.